api/test/shell/cli_test.sh

```diff
@@ -126,6 +126,28 @@ if [[ $res != "hi~" ]]; then
 fi
 clean_up
 
+# run with -p flag out of the default directory
+workDir=$(pwd)
```

Comment:
Looks like it's still in default dir. could we mv it to another dir to test it ?

Reply:
@nic-chen From my point of view, the default directory only makes sense for `manager-api` binary. But it's indeed not intuitive. I would renew this test case. Thanks.

Reply:
oh, yes, that make sense.

Reply:
@nic-chen Hi, PR renewed with a more intuitive way. Thanks for reviewing. 😄